discord/ext/commands/core.py

```diff
@@ -657,6 +657,15 @@ def clean_params(self) -> Dict[str, inspect.Parameter]:
         """
         return self.params.copy()
 
+    @property
+    def cooldown(self) -> Optional[Cooldown]:
+        """Optional[:class:`Cooldown`]: The cooldown of a command when invoked or ``None`` if the command
+        doesn't have a registered cooldown.
+        
```

Comment:
This has trailing whitespace which fails black.

Reply:
Right, sorry. Should be fine to merge now?

Reply:
The trailing whitespace is still there.

Reply:
Fixed